v2.0/transactions.md

```diff
@@ -219,27 +219,24 @@ The client can also display the current isolation level of the transaction with
 
 With `SERIALIZABLE` isolation, a transaction behaves as though it has the entire database all to itself for the duration of its execution. This means that no concurrent writers can affect the transaction unless they commit before it starts, and no concurrent readers can be affected by the transaction until it has successfully committed. This is the strongest level of isolation provided by CockroachDB and it's the default.
 
-Unlike `SNAPSHOT`, `SERIALIZABLE` isolation permits no anomalies. However, due to CockroachDB's transaction model, `SERIALIZABLE` isolation may require more transaction restarts, especially in the presence of high contention between concurrent transactions. Consider using `SNAPSHOT` isolation for high contention workloads.
+Unlike `SNAPSHOT`, `SERIALIZABLE` isolation permits no anomalies. `SERIALIZABLE` isolation may require transaction restarts to prevent [write skew](https://en.wikipedia.org/wiki/Snapshot_isolation) anomalies.
```

Comment:
Elsewhere, we say:

> Unlike the `SERIALIZABLE` level, `SNAPSHOT` isolation permits the write skew anomaly. 

Here we suggest that write skew can happen with `SERIALIZABLE`, which feels contradictory?

Reply:
Re-worded to make this clear. It prevents `write skew` by noticing it's possible and requiring restarts.